docs/products/kafka/howto/datadog-customised-metrics.rst

```diff
@@ -42,26 +42,44 @@ Before customising the metrics, make sure that you have a Datadog endpoint confi
 To customise the metrics sent to Datadog, you can use the ``service integration-update`` passing the following customised parameters:
 
 * ``kafka_custom_metrics``: defining the comma separated list of custom metrics to include (within ``kafka.log.log_size``, ``kafka.log.log_start_offset`` and ``kafka.log.log_end_offset``)
+
+As example to sent the ``kafka.log.log_size`` and ``kafka.log.log_end_offset`` metrics execute the following code:
+
+.. code::
+
+    avn service integration-update                                                \
+        -c kafka_custom_metrics=['kafka.log.log_size','kafka.log.log_end_offset'] \
+        INTEGRATION_ID
+
+Once the update is successful and metrics have been collected and pushed, you should see them in your Datadog explorer.
+
+.. seealso:: Learn more about :doc:`/docs/integrations/datadog`.
+
+
+Customise Apache Kafka® Consumer Integration metrics sent to Datadog
```

Comment:
Do you think we can shorten the heading a bit, say something like: 

Customize  Apache Kafka® consumer metrics for Datadog
-----------------------------------------------------------